### PR TITLE
Docs: Adding a section for making design contributions to Gestalt

### PR DESCRIPTION
**Pull request by ponori**
Added a section to our How To Work With Us page detailing design contributions.

### CODE REVIEW

File: docs/pages/how_to_work_with_us.js [hunk 1]
```diff
@@ -67,6 +67,37 @@ We ask that any ideas that change the functionality of an existing Gestalt compo
...
+    - Sign up for an Office Hours slot! Explain project goals and show multiple options explored with and without Gestalt components to help us understand your needs. This does not need to be a polished presentation, but should have enough detail that we understand the ask and why it is needed.
+    - Coordinate with other designers if there are overlapping needs to help us better prioritize in our roadmap.
+    - Be able to explain how this component should be used or not used. This helps us in testing the component against existing components and product surfaces.
+    - Have a rough project timeline available. This allows us to determine if we’re able to accommodate the work.
+2. **Create a Branch in our main file following the steps**
+    OK, great. You went to office hours and we all agree on what we're making. Now it's time to start bringing your designs into Gestalt.
+
+    - Create a Branch file in our main file. New to Branching? Don't worry, our design team will support you with that.
+    - Follow the name convention: Your initials + Component name +What you are doing, for example: CR - Tab: Adding a badge variant.
```
Comment (cintiapub) [inline docs/pages/how_to_work_with_us.js:85]: _Follow the name convention: Your initials + Component name +What you are doing, for example: CR - Tab: Adding a badge variant._ 

Minor minor thing: Just add a space after the "+" on "What you are doing".

It looks great! Thanks for the addition!

**Review by cintiapub — APPROVED** [review]
The copy looks good to me. I made a comment (minor minor thing to fix), otherwise it looks good! Thank you.

File: docs/pages/how_to_work_with_us.js (hunk 1 above)
Comment (ponori) [reply docs/pages/how_to_work_with_us.js]: Good catch! Done!

File: docs/pages/how_to_work_with_us.js [hunk 2]
```diff
@@ -67,6 +67,37 @@ We ask that any ideas that change the functionality of an existing Gestalt compo
   </MainSection>,
 );
 
+card(
+  <MainSection name="How can you make design contributions to Gestalt?">
```
Comment (ayeshakmaz) [inline docs/pages/how_to_work_with_us.js:71]: Now that we have 3 pieces about contributing, maybe it'd help to organize them under a single MainSection called "Contributing", and then detail it with like subsections like  "Overall process", "Design changes", "Engineering changes" or something like that to create a little more organization?

Comment (ayeshakmaz) [reply docs/pages/how_to_work_with_us.js:71]: Actually maybe 4 subsections, adding one for "Other ways to contribute"

Comment (ponori) [reply docs/pages/how_to_work_with_us.js]: OK, so I kept the `What is the process to request new additions or changes?` as a separate section as it's a little hard to unfurl the specifics of contributions in that section. 

The other three sections I popped into a `How can you contribute to Gestalt?` section. 

Let me know what you think!

**Review by ayeshakmaz — APPROVED** [review]

**Pull request merged**